### DISCUSSION TITLE
Custom entrypoint for container jobs

### DISCUSSION THREAD

**Discussion by chuckatkins** [community/community#26465] [Actions]
<p>Is there a way I can define the entrypoint and startup command used by a container job?  It  From lookingat the “Initialize Containers” step shows the container always getting created with <code>--entrypoint "tail"</code>, evn if I pass a differnt entrypoint in the create actions then the “tail” entrypoint is still added after.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/chuckatkins">@chuckatkins</a> , </p>
<p>Using custom entrypoint and startup command for a container job is not supported. </p>
<p>The <strong> </strong>** --entrypoint “tail” &lt;image name&gt; “-f” “/dev/null”** in “Initialize Containers” step  is to keep the container running. </p>
<p>If you want to run some startup command, I would recommend you adding a run script step as the first step of the job. </p>
